### PR TITLE
Add Draft LoRA scripts

### PR DESCRIPTION
**Pull request by shuaills**
## Motivation

This PR added training scripts for LoRA. This is a draft PR, do not merge.
## Benchmark & Profiling

Acc on W/WO LoRA
image

## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://sgl-fru7574.slack.com/archives/C09784E3EN6 to discuss your PR.

### CODE REVIEW

**Comment by zyksir** [comment]
cc @shuaills  is still working on this and this PR is not ready yet. I will leave it open

**Comment by ggg-s** [comment]
I added LoRA to the Q/V matrices, but it performed poorly. The loss stuck at 0.8. Is the trainable parameter count too low?
